src/css/popup.css

```diff
@@ -1,3 +1,111 @@
+* {
```

Comment:
this is trading minimized bootstrap for 130 (plus more in future) lines of custom code we have to maintain. Unfortunately it doesn't add much that i can see as far as styling. Let's not do this please.

Reply:
Yes, but the 130 lines are simple and _only_ what we need. This allows us to control exactly what the CSS is rather than having to worry about an additional dependency. 

All of this is extracted directly out of the main bootstrap styles, with any changes being the styles that we're customizing.

There is minimal need for any browser-specific support here, as all our styles are supported widely.

Reply:
there's no worry about an additional dependency. please don't touch bootstrap, there's no issue with it that we are trying to fix.